tests/foreman/ui/test_host.py

```diff
@@ -1878,6 +1879,45 @@ def test_all_hosts_bulk_delete(target_sat, function_org, function_location, new_
         assert session.all_hosts.bulk_delete_all()
 
 
+@pytest.mark.tier2
+def test_all_hosts_bulk_cve_reassign(
```

Comment:
Btw, do we aim to run this test case against all rhel versions, or is it fine to use just the default one?
```suggestion
@pytest.mark.tier2
@pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
def test_all_hosts_bulk_cve_reassign(
```

Reply:
it should be this way, I'm not sure why that didn't make it in